configs/settings-example.yml

```diff
@@ -0,0 +1,131 @@
+# Variables for ansible provisioning
+# See https://docs.ansible.com/ansible/latest/playbooks_variables.html
+# https://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable
+env: "dev"
```

Comment:
Let's ditch this, it's used only twice. I've learned it's a bad practise to have env flags like this, and we should have feature flags instead (like we do, e.g. `debug:true` or `cache:true`).

https://github.com/Hitchwiki/hitchwiki/blob/master/configs/mediawiki.php#L57

That section can be like this instead:
```php

// Enable error reporting
if ($hwDebug) {
  error_reporting( -1 );
  ini_set( 'display_errors', 1 );
  
  // Show the debug toolbar if `hw_debug` is set on the request,
  // either as a parameter or a cookie.
  // https://www.mediawiki.org/wiki/Debugging_toolbar
  if ( !empty( $_REQUEST['hw_debug'] ) ) {
    $wgDebugToolbar = true;
  }
}

// Expose debug info for PHP & SQL errors.
$wgShowExceptionDetails = $hwDebug;
$wgDevelopmentWarnings = $hwDebug;
$wgDebugDumpSql = $hwDebug;
$wgShowDBErrorBacktrace = $hwDebug;
$wgShowSQLErrors = $hwDebug;
$wgResourceLoaderDebug = $hwDebug;

// Profiling
$wgDebugProfiling = false;

// Log into file
$logDir = '/vagrant/logs'; //TODO, 
$wgDebugLogFile = "{$logDir}/mediawiki-debug.log";
foreach ( array( 'exception', 'runJobs', 'JobQueueRedis' ) as $logGroup ) {
  $wgDebugLogGroups[$logGroup] = "{$logDir}/mediawiki-{$logGroup}.log";
}
```

https://github.com/Hitchwiki/hitchwiki/blob/ba64070be06999c43410f5d56fc2fad1bccd8566/public/robots.txt.php#L12

I'll write more about robots.txt later.

Reply:
(Can be dealt also later on, though!)

Reply:
php is not my greatest fun, i leave this up to you :) #166